### PR TITLE
wg_engine: fix stroke trimming

### PR DESCRIPTION
**Pull request by mgrudzinska**
Handle very close values of begin and end.

### CODE REVIEW

File: src/renderer/wg_engine/tvgWgGeometry.h [hunk 1]
```diff
@@ -124,15 +124,15 @@ struct WgVertexBuffer {
     }
 
     // append source vertex buffer in index range from start to end (end not included)
-    void appendRange(const WgVertexBuffer& buff, size_t start_index, size_t end_index) {
-        if (start_index <= end_index)
-            for (size_t i = start_index; i < end_index; i++)
-                append(buff.vbuff[i]);
-        if (start_index > end_index) {
+    void appendRange(const WgVertexBuffer& buff, size_t start_index, size_t end_index, bool loop) {
```
Comment (SergeyLebedkin) [inline src/renderer/wg_engine/tvgWgGeometry.h:127]: make a loop value false by default

Comment (SergeyLebedkin) [inline src/renderer/wg_engine/tvgWgGeometry.h:127]: make a loop value false by default

Comment (SergeyLebedkin) [reply src/renderer/wg_engine/tvgWgGeometry.h:127]: So please, at least mention it in comments

Comment (mgrudzinska) [reply src/renderer/wg_engine/tvgWgGeometry.h:127]: @SergeyLebedkin please check https://github.com/thorvg/thorvg/pull/2771, should be more general

**Comment by mgrudzinska** [comment]
https://github.com/thorvg/thorvg/pull/2771

**Pull request closed without merging**